src/components/Atoms/Alert/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import BREAKPOINT from "../../../variables/breakpoint";
+import FONTFAMILY from "../../../variables/font_family";
+import TEXT from "../../../variables/texts";
+
+const TextAlert = styled.div`
+  color: ${COLOR.WHITE};
+  font-family: ${FONTFAMILY.NOTO_SANS};
+  ${TEXT.S};
+  line-height: 40px;
+`;
+
+/*テキスト表示*/
+export const Alert = ({ isActive, isActiveText }) => {
+  return (
+    <DisplayAlert isActive={isActive}>
+      <TextAlert isActive={isActive}>{isActiveText}</TextAlert>
+    </DisplayAlert>
+  );
+};
+
+const DisplayAlert = styled.div`
+  opacity: ${(props) => (props.isActive ? 1 : 0)};
+  transition: 0.5s;
+  transform: translate(-50%, 0)
```

Comment:
これだとどうなるだろう...？（うまくいくかは保証できない💦）
```suggestion
  transform: translate(0px, -40px)
```